### ISSUE TITLE
Colorize Preview Tooltips Too

### ISSUE THREAD

**Issue by the-a-man-006** [kitamstudios/rust-analyzer.vs#28]
Apply code colorization to the code items that show up in the preview tooltips that pop up upon hovering the cursor on code items. Also, optionally, copy the color scheme that VisualAssist uses (lots of yellows, golds, orange, and light-red and green (on comments, iirc), with no sharp contrast jumps from word to word; superior).

**Comment by parthopdas** [comment]
thanks. please can you paste images so i know what you are referring to.

**Comment by the-a-man-006** [comment]
![vs_rust_coloring](https://github.com/kitamstudios/rust-analyzer.vs/assets/58506175/cd472187-4410-4067-a427-116115d722bb)

**Comment by parthopdas** [comment]
yeah this will be very nice. please can you file a bug on visual studio to support color tooltips for RLS extensions? they do not have that yet hence you see the b/w tooltip. the tooltip and its data are exchanged directly between RLS and VS, r-a.vs doesn't get to handle them.

same scenario on vscode looks pretty for sure
![image](https://github.com/kitamstudios/rust-analyzer.vs/assets/2140460/e2d79997-64cc-4120-b2b9-55fa9cfceef5)

**Comment by parthopdas** [comment]
Will require a shim layer between r-a.vs & ra.
